components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/internal/CaptchaDataHolder.java

```diff
@@ -168,6 +171,16 @@ public void setReCaptchaScoreThreshold(double reCaptchaScoreThreshold) {
         this.reCaptchaScoreThreshold = reCaptchaScoreThreshold;
     }
 
+    public double getReCaptchaWarnScoreThreshold() {
```

Comment:
Shall we add method comments?

Reply:
Update PR sent: https://github.com/wso2-extensions/identity-governance/pull/809